event-log/src/test/scala/io/renku/eventlog/init/EventStatusRenamerImplSpec.scala

```diff
@@ -0,0 +1,120 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.eventlog.init
+
+import cats.effect.IO
+import ch.datascience.generators.Generators.Implicits._
+import ch.datascience.graph.model.events.{EventBody, EventId, EventStatus}
+import ch.datascience.graph.model.events.EventStatus._
+import ch.datascience.interpreters.TestLogger
+import ch.datascience.interpreters.TestLogger.Level.Info
+import doobie.implicits._
+import io.circe.literal.JsonStringContext
+import io.renku.eventlog.DbEventLogGenerators._
+import eu.timepit.refined.auto._
+import io.renku.eventlog.{CompoundId, Event, EventLogDataFetching, EventLogDataProvisioning, InMemoryEventLogDb}
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpec
+
+class EventStatusRenamerImplSpec
+    extends AnyWordSpec
+    with DbInitSpec
+    with should.Matchers
+    with EventLogDataProvisioning
+    with EventLogDataFetching {
+  protected override lazy val migrationsToRun: List[Migration] = List(
+    eventLogTableCreator,
+    projectPathAdder,
+    batchDateAdder,
+    latestEventDatesViewRemover,
+    projectTableCreator,
+    projectPathRemover,
+    eventLogTableRenamer
+  )
+
+  "run" should {
+    s"rename all the events from PROCESSING to GENERATING_TRIPLES " in new TestCase {
+      val processingEvents = events.generateNonEmptyList(minElements = 2)
+      processingEvents.map(event => store(event, withStatus = "PROCESSING"))
+      val otherEvents = events.generateNonEmptyList()
+      otherEvents.map(event => store(event, withStatus = event.status.toString))
+
+      eventStatusRenamer.run().unsafeRunSync() shouldBe ((): Unit)
+
+      findEvents(status = GeneratingTriples).eventIdsOnly.toSet shouldBe processingEvents
+        .map(_.compoundEventId)
+        .toList
+        .toSet
+
+      logger.loggedOnly(Info(s"'PROCESSING' event status renamed to 'GENERATING_TRIPLES'"))
+    }
+
+    s"Not to anything if there are no events with the status PROCESSING" in new TestCase {
+      val otherEvents = events.generateNonEmptyList()
+      otherEvents.map(event => store(event, withStatus = event.status.toString))
+
+      eventStatusRenamer.run().unsafeRunSync() shouldBe ((): Unit)
+
+      findEvents(status = GeneratingTriples).eventIdsOnly.toSet shouldBe Set.empty[CompoundId]
+
+      findEventsId shouldBe otherEvents.map(_.id).toList.toSet
+
+      logger.loggedOnly(Info(s"'PROCESSING' event status renamed to 'GENERATING_TRIPLES'"))
+    }
+
+  }
+
+  private trait TestCase {
+    val logger             = TestLogger[IO]()
+    val eventStatusRenamer = new EventStatusRenamerImpl[IO](transactor, logger)
+  }
+
+  private def store(event: Event, withStatus: String): Unit = {
+    upsertProject(event.compoundEventId, event.project.path, event.date)
+    execute {
```

Comment:
Sadly I could not use the method in the EventLogDataProvisioning trait as I removed the Processing EventStatus, I needed to pass in a String